_includes/extensions-form.md

```diff
@@ -0,0 +1 @@
+Already built an extension? [Let us know about your experience](https://docs.google.com/forms/d/e/1FAIpQLSenjK8KiBiOrwCpq06u_iVEZGv4tfTEpfmY_cQDmsqJrHL9Gw/viewform).
```

Comment:
It looks like we're starting to use such feedback forms more frequently; these URLs will likely be temporary, and removed after some time.

I'm wondering if we should start using "permalink" redirects for these, like we do for the `/go/` redirects that we use when linking from our products; https://github.com/docker/docs/blob/380bf39b943201da19b537a71872c7a718d60922/_redirects.yml#L79-L93.

If we don't want to "pollute" the `/go/` namespace too much with such temporary URLs, we could consider to use another prefix (`/feedback/xxxx`) to be slightly clearer on intent, although `/go/` should still be an option, as long as it's somewhat clear what the purpose is/was; e.g. `/go/extensions-sdk-feedback/`.

Using those "friendly URL" redirects allows for;

- The URLs to be easy-to-use / communicate (`docs.docker.com/go/extensions-sdk-feedback/` is easier to remember than `docs.google.com/forms/<some very long ID>`)
- Once the form is no longer used, we can redirect the URLs to elsewhere (which could be just the extensions SDK overview page).
- If we need a new feedback form in future for the same purpose, we can re-use the old redirect to point to the new feedback form.

Of course we cannot prevent users referencing the actual (docs.google.com....) URL, but as long as we don't communicate the direct URL, and only communicate the "friendly" redirect-URL, that will be less common.

Reply:
Let me quickly ping @crazy-max to check if using a different (e.g. `/feedback/`) prefix would complicate things from a technical perspective (if we would decide to use that instead of `/go/`). Also wondering if "sub-directories" (`/go/feedback/<some easy name>`) would work with our current implementation (that way we keep things inside the `/go/` "namespace", but still have a bit of separation between "feedback" and other kind of redirects).

Reply:
I think `/feedback/` is more explicit :+1:

Can be done like samples:

```yaml
# feedback
"https://docs.google.com/forms/d/e/1FAIpQLSenjK8KiBiOrwCpq06u_iVEZGv4tfTEpfmY_cQDmsqJrHL9Gw/viewform":
  - /feedback/extension/
```

Reply:
Thank you @thaJeztah, this is a much neater way of doing it and the benefits are clear. Have edited the PR accordingly 😃